### ISSUE TITLE
Identifying mismatch between training and validation set

### ISSUE THREAD

**Issue by ivanzvonkov** [nasaharvest/crop-mask#119]
**Context:** To obtain an unbiased metric for each generated map, a separate classifier is trained for each region of interest with a distinct validation and test set (specific to that region).

**Issue:** Though a similar method is followed for each region, metrics on different validation sets have significant fluctuations.

**Potential solution:** Identifying the discrepancy between training and validation sets may help us better understand the fluctuations in metrics across regions of interest. One way of measuring discrepancy is detailed here: http://ciml.info/dl/v0_99/ciml-v0_99-ch08.pdf

Another potential solution is to experiment with the deepchecks library: https://github.com/deepchecks/deepchecks
